deeppavlov/skills/odqa/README.md

```diff
@@ -0,0 +1,139 @@
+and the [reader config](https://github.com/deepmipt/DeepPavlov/tree/master/deeppavlov/models/squad#config-components) should be used for reader training.
+
+### Ranker
+
+The ranker config for **English** language can be found at `deeppavlov/configs/odqa/en_ranker_prod.json`
+
+The ranker config for **Russian** language can be found at `deeppavlov/configs/odqa/ru_ranker_prod.json`
+
+* **dataset_iterator** - downloads Wikipidia DB, creates batches for ranker fitting
+    * **_data_dir_** - a directory to download DB to
+    * **_data_url_** - an URL to download Wikipedia DB from
+    * **_shuffle_** - whether to perform shuffling when iterating over DB or not
+* **chainer** - pipeline manager
+    * **_in_** - pipeline input data (questions)
+    * **_out_** - pipeline output data (Wikipedia articles ids and scores of the articles)
+* **tfidf_ranker** - the ranker class
+    * **_in_** - ranker input data (questions)
+    * **_out_** - ranker output data (Wikipedia articles ids)
+    * **_fit_on_batch_** - fit the ranker on batches of Wikipedia articles
+    * **_vectorizer_** - a vectorizer class
+        * **_fit_on_batch_** - fit the vectorizer on batches of Wikipedia articles
+        * **_save_path_** - a path to serialize a vectorizer to
+        * **_load_path_** - a path to load a vectorizer from
+        * **_tokenizer_** - a tokenizer class
+            * **_lemmas_** - whether to lemmatize tokens or not
+            * **_ngram_range_** - ngram range for vectorizer features
+* **train** - parameters for vectorizer fitting
+    * **_validate_best_**- is ingnored, any value
+    * **_test_best_** - is ignored, any value
+    * **_batch_size_** - how many Wikipedia articles should return the dataset iterator in a single batch
+
+### ODQA
+
+Default ODQA config for **English** language is `deeppavlov/configs/odqa/en_odqa_infer_prod.json`
+
+Default ODQA config for **Russian** language is `deeppavlov/configs/odqa/en_odqa_infer_prod.json`
+
+The components of ODQA config can be referred to [ranker config](#the-ranker-config)
+and [reader config](https://github.com/deepmipt/DeepPavlov/tree/master/deeppavlov/models/squad#config-components) accordingly.
+However, main inputs and outputs are worth explaining:
+
+* **chainer** - pipeline manager
+    * **_in_** - pipeline input data (questions)
+    * **_out_** - pipeline output data (answers)
+
+## Pretrained models
+
+Wikipedia data and pretrained ODQA models are downloaded in `deeppavlov/download/odqa` by default.
+
+### enwiki.db
+
+**enwiki.db** SQLite database consists of **5159530 Wikipedia articles**
+and is built by the following steps:
+1. Download a Wikipedia dump file. We took the latest [enwiki](https://dumps.wikimedia.org/enwiki/20180201)
+ (from 2018-02-11)
+2. Unpack and extract the articles with [WikiExtractor](https://github.com/attardi/wikiextractor)
+ (with `--json`, `--no-templates`, `--filter_disambig_pages` options)
+3. Build a database.
```

Comment:
Looks like this block should either say less (that .db was built from this dump and that's it) or explain how to build a database in more detail.

Reply:
Agree.